applications/IgaApplication/custom_modelers/iga_modeler.cpp

```diff
@@ -0,0 +1,279 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License 
+//                   Kratos default license: kratos/license.txt
+//
+
+
+// Project includes
+#include "iga_modeler.h"
+
+
+namespace Kratos
+{
+    ///@name Stages
+    ///@{
+
+    void IgaModeler::SetupModelPart()
+    {
+        KRATOS_ERROR_IF_NOT(mParameters.Has("cad_model_part_name"))
+            << "Missing \"cad_model_part\" section" << std::endl;
+        ModelPart& cad_model_part = mpModel->GetModelPart(mParameters["cad_model_part_name"].GetString());
+        KRATOS_ERROR_IF_NOT(mParameters.Has("analysis_model_part_name"))
+            << "Missing \"analysis_model_part_name\" section" << std::endl;
+        ModelPart& analysis_model_part = mpModel->GetModelPart(mParameters["analysis_model_part_name"].GetString());
+
+        const std::string& rDataFileName = mParameters.Has("physics_file_name")
+            ? mParameters["physics_file_name"].GetString()
+            : "physics.iga.json";
```

Comment:
Does it make sense to also check if there is a physics file? Maybe doing something like it is done for the cad_model_part and analysis_model_part: 
```C++
KRATOS_ERROR_IF_NOT(mParameters.Has("physics_file_name"))
```

Reply:
Basically, that's what I am trying with this if call. When `"physics_file_name"` is not provided it considers it being `"physics.iga.json"`. I know that this is not fully bullet proof. But we'll need to gain a little bit of experience of how this will be called.